src/Auth/Rules/Rule.php

```diff
@@ -21,7 +21,6 @@
      * @param string $role role of the user
      * @param Request $request current request, used to get a default table if not provided
      * @return bool
-     * @throws \OutOfBoundsException if table is not found or it doesn't have the expected fields
```

Comment:
I feel like this belongs to the abstract class since custom logic may or may throw any exceptions and we are not catching any specific ones